src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -232,6 +232,14 @@ void java_bytecode_convert_method_lazy(
   method_symbol.mode=ID_java;
   method_symbol.location=m.source_location;
   method_symbol.location.set_function(method_identifier);
+  if(m.is_public)
+    member_type.set(ID_access, ID_public);
```

Comment:
the problem is that `set_access` is defined for `struct_union_typet` and therefore is only really usable for fields here. I had thought about using a special `java_typet` derived from `typet` which could have the `set_access`, too. Would you prefer that?